hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -343,6 +343,7 @@ private OzoneConsts() {
 
   // Default OMServiceID for OM Ratis servers to use as RaftGroupId
   public static final String OM_SERVICE_ID_DEFAULT = "omServiceIdDefault";
+  public static final String OM_DEFAULT_NODE_ID = "om1";
 
   // Dummy OMNodeID for OM Clients to use for a non-HA OM setup
   public static final String OM_NODE_ID_DUMMY = "omNodeIdDummy";
```

Comment:
I assume this `omNodeIdDummy` will only be use if someone disables Ratis?

Reply:
No, this would be used for single node Ratis also.

Reply:
Sorry I didn't understand then - how do we use both `OM_DEFAULT_NODE_ID` and `OM_NODE_ID_DUMMY ` with single node Ratis?

Reply:
OM_NODE_ID_DUMMY was being used just as a dummy nodeId in OMFailoverProxyProvider for non-HA cluster.
But now that we have a default, we should remove the dummy. It would not affect the client in anyway as this is only for the non-HA and also. 
Thanks for pointing this out Arpit. I will update the PR.